airflow/providers/amazon/aws/hooks/elasticache.py

```diff
@@ -0,0 +1,265 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from time import sleep
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class ElastiCacheHook(AwsBaseHook):
+    """
+    Interact with AWS ElastiCache
+    """
+
+    # Constants for ElastiCache describe API response keys
+    REPLICATION_GROUPS = 'ReplicationGroups'
+    REPLICATION_GROUP = 'ReplicationGroup'
+    STATUS = 'Status'
+
+    # Constants for ElastiCache replication group status
+    STATUS_CREATING = 'creating'
+    STATUS_AVAILABLE = 'available'
+    STATUS_DELETING = 'deleting'
+    STATUS_CREATE_FAILED = 'create-failed'
+    STATUS_MODIFYING = 'modifying'
+    STATUS_SNAPSHOTTING = 'snapshotting'
```

Comment:
```suggestion
```

These constants make it harder to understand the code and isn't the style we use in Airfow, please use the string literals directly.